openQA.spec

```diff
@@ -45,24 +45,32 @@
 %else
 %define python_scripts_requires %{nil}
 %endif
-%define assetpack_requires perl(Mojolicious::Plugin::AssetPack) >= 1.36, perl(CSS::Minifier::XS) perl(JavaScript::Minifier::XS)
-%define common_requires perl(Archive::Extract) perl(Config::IniFiles) perl(Cpanel::JSON::XS) perl(Cwd) perl(Data::Dump) perl(Data::Dumper) perl(Digest::MD5) perl(Getopt::Long) perl(Minion) >= 10.04, perl(Mojolicious) >= 8.24, perl(Try::Tiny) perl(Regexp::Common), perl(Storable)
+# The following line is generated from dependencies.yaml
+%define assetpack_requires perl(CSS::Minifier::XS) >= 0.01 perl(JavaScript::Minifier::XS) >= 0.11 perl(Mojolicious::Plugin::AssetPack) >= 1.36
```

Comment:
Last time I checked the commas to separate versioned tokens were necessary. Do you know how this works now? I can see https://build.opensuse.org/package/show/devel:openQA:TestGithub:PR-2934/openQA is fine but still :)

Reply:
I tested this syntax with another package in OBS, and also increased the version to check if it would correctly return unresolvable, and it did. so it seems to work.